lib/lookup.json

```diff
@@ -457,25 +343,12 @@
     "envVar": { "npm_config_ignore_scripts": "true" },
     "comment": "postlint script fails"
   },
-  "serialport": {
-    "prefix": "serialport@",
-    "flaky": ["ppc", "rhel"],
-    "tags": "native",
-    "maintainers": "reconbot",
-    "skip": ["win32"]
-  },
   "socket.io": {
     "maintainers": "rauchg",
     "head": true,
     "skip": ">=17",
     "comment": "WS is not compatible"
   },
-  "spawn-wrap": {
```

Comment:
Yes, remove. Not actively maintained.

Reply:
Thanks. Removed in https://github.com/nodejs/citgm/pull/965.